### PR TITLE
Added workflow for operator PR's

### PR DESCRIPTION
**Pull request by sabre1041**
Added workflow for operator PR's

### CODE REVIEW

File: .github/workflows/pr-operator.yml [hunk 1]
```diff
@@ -0,0 +1,400 @@
...
+  workflow_call:
+    inputs:
+      OPERATOR_SDK_VERSION:
+        description: "Version of Operator SDK Binary for installation"
+        default: "v1.9.0"
+        required: false
+        type: string
+      BUILD_PLATFORMS:
+        description: 'Comma separated list of targets for builds e.g. "linux/amd64,linux/arm64,linux/ppc64le"'
+        default: "linux/amd64,linux/arm64,linux/ppc64le"
```
Comment (raffaelespazzoli) [inline .github/workflows/pr-operator.yml:12]: please add z/OS here in the default. The issue that was preventing us from building zos images should have been resolved.

File: .github/workflows/pr-operator.yml [hunk 2]
```diff
@@ -0,0 +1,400 @@
...
+      - name: Build Go Cache Paths
+        id: go-cache-paths
+        run: |
+          echo "::set-output name=go-build::$(go env GOCACHE)"
+          echo "::set-output name=go-mod::$(go env GOMODCACHE)"
+
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v1
+        with:
+          go-version: ^1.16
```
Comment (raffaelespazzoli) [inline .github/workflows/pr-operator.yml:82]: can we make this an input variable, with a default instead of a hard-coded value?

**Review by raffaelespazzoli — CHANGES_REQUESTED** [review]

File: .github/workflows/pr-operator.yml (hunk 1 above)
Comment (sabre1041) [reply .github/workflows/pr-operator.yml]: Addressed

File: .github/workflows/pr-operator.yml (hunk 2 above)
Comment (sabre1041) [reply .github/workflows/pr-operator.yml]: Added

**Review by nickjordan — APPROVED** [review]
os/arch and input changes LGTM

**Pull request merged**